libs/angular/src/lib/slider/slider.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="ngg-slider-label-container">
+  <label [attr.for]="name">Slider label text in one line</label>
```

Comment:
Label text should probably be an Input? :)

Reply:
Totally missed this. Will change this one too!